utils.py

```diff
@@ -390,6 +390,15 @@ def hasAutobake(props):
             return True
     return False
 
+def hasGalera(props):
+    builderName = str(props.getProperty("buildername"))
+
+    for b in builders_galera:
+        if builderName in b:
+            return True
+    return False
+
+@util.renderer
```

Comment:
is `@util.renderer` needed?

Reply:
I don't know. I haven't worked out the approach of when its needed and when not. Things like `hasCompat` don't have them but `hasBigtest` does. I'm leaning towards doesn't need (and hasBigtest doesn't need it either). Back to documentation for me unless you have some good insights for this.